cmd/crowdsec-cli/bouncers.go

```diff
@@ -54,55 +108,11 @@ Note: This command requires database direct access, so is intended to be run on
 		Args:              cobra.ExactArgs(0),
 		DisableAutoGenTag: true,
 		Run: func(cmd *cobra.Command, arg []string) {
-			blockers, err := dbClient.ListBouncers()
+			bouncers, err := getBouncers(dbClient)
```

Comment:
in csv/human mode, `getBouncers` displays table or write csv, but still returns the bytes it read. It will trigger double display in csv/human ?